lerobot/common/datasets/factory.py

```diff
@@ -35,11 +35,18 @@ def resolve_delta_timestamps(cfg):
                 cfg.training.delta_timestamps[key] = eval(delta_timestamps[key])
 
 
-def make_dataset(
-    cfg,
-    split="train",
-):
-    if cfg.env.name not in cfg.dataset_repo_id:
+def make_dataset(cfg, split: str = "train") -> LeRobotDataset | MultiLeRobotDataset:
+    """
+    Args:
+        cfg: A Hydra config as per the LeRobot config scheme.
+        split: TODO(now)
```

Comment:
@Cadene I'm not sure what this actually does (in terms of the way we are using it now). Can you please suggest a docstring for it.

Reply:
```suggestion
        split: Select the data subset used to create an instance of LeRobotDataset.
                 All datasets hosted on [lerobot](https://huggingface.co/lerobot) contain only one subset: "train".
                 Thus, by default, `split="train"` selects all the available data. `split` aims to work like the slicer
                 in the hugging face datasets: https://huggingface.co/docs/datasets/v2.19.0/loading#slice-splits
                 As of now, it only supports `split="train[:n]"` to load the first n frames of the dataset or 
                 `split="train[n:]"` to load the last n frames. For instance `split="train[:1000]"`.
```

Reply:
@Cadene thanks for this. I'll accept it but as two side notes for later:

- This may be a little involved to go in a parameter description. Perhaps it should live in a more prominent location and this docstring can point there if needed.
- Ideally these docs go in LeRobotDatset and MultiLeRobotDataset! But that's not pertinent  to this PR.

Reply:
It lives already in LeRobotDataset